runners/cli/src/main/kotlin/cli/main.kt

```diff
@@ -1,339 +1,344 @@
 package org.jetbrains.dokka
 
+import kotlinx.cli.*
 import org.jetbrains.dokka.DokkaConfiguration.ExternalDocumentationLink
 import org.jetbrains.dokka.utilities.DokkaConsoleLogger
-import java.io.File
-import java.io.FileNotFoundException
+import org.jetbrains.kotlin.utils.addToStdlib.cast
+import java.io.*
 import java.net.MalformedURLException
 import java.net.URL
-import java.net.URLClassLoader
 import java.nio.file.Files
 import java.nio.file.Paths
 
-open class GlobalArguments(parser: DokkaArgumentsParser) : DokkaConfiguration {
-    override val outputDir: String by parser.stringOption(
-        listOf("-output"),
-        "Output directory path",
-        ""
-    )
+class GlobalArguments(args: Array<String>) : DokkaConfiguration {
+
+    val parser = ArgParser("globalArguments", prefixStyle = ArgParser.OptionPrefixStyle.JVM)
+
+    override val outputDir by parser.option(ArgType.String, description = "Output directory path").required()
 
-    override val format: String by parser.stringOption(
-        listOf("-format"),
-        "Output format (text, html, gfm, jekyll, kotlin-website)",
-        ""
+    override val format by parser.option(
+        ArgType.String,
+        description = "Output format (text, html, gfm, jekyll, kotlin-website)"
```

Comment:
We only have `html`, `gfm` and `jekyll` afaik

Reply:
Salvaged from old one, but will fix